### ISSUE TITLE
PEPOLAR field map can't be recognised after upgrading to version 22.0.0

### ISSUE THREAD

**Issue by manzouri** [nipreps/fmriprep#2848]
### What happened?

I could use my PEPOLAR filed maps with earlier version but v 22 can't recognise them any Moreland ignore field maps

### What command did you use?

```shell
fmriprep-docker $bids_root_dir $out_dir/ \
    participant \
    --participant-label $subject \
    --skip-bids-validation \
    --md-only-boilerplate \
    --output-spaces MNI152NLin2009cAsym \
    --fs-no-reconall \
    --fs-license-file /usr/local/freesurfer/license.txt \
```

### What version of fMRIPrep are you running?

22.0.0

### How are you running fMRIPrep?

Docker

### Is your data BIDS valid?

Yes

### Are you reusing any previously computed results?

No

### Please copy and paste any relevant log output.

```shell
It runs smoothly but without field maps
```

**Comment by mgxd** [comment]
Can you remove `--skip-bids-validation` and add `-vv` to your command and retry? This will give us more information on what's happening.

**Comment by manzouri** [comment]
It runs smoothly and generates the report without error but without field maps. I attach the log file here.
[report.csv](https://github.com/nipreps/fmriprep/files/9551211/report.csv)

**Comment by manzouri** [comment]
Hi @mgxd , did you get any chance to look at this?

**Comment by mgxd** [comment]
Yeah I took a look, but nothing in the report jumps out. Perhaps you can share the directory tree of the subject you are running into trouble with, along with the relevant fieldmap metadata (`IntendedFor` or `B0FieldIdentifier` fields).

**Comment by manzouri** [comment]
Here is the tree: 
├── dataset_description.json
└── sub-102
    └── ses-01
        ├── anat
        │   ├── sub-102_ses-01_run-01_T1w.json
        │   ├── sub-102_ses-01_run-01_T1w.nii.gz
        │   ├── sub-102_ses-01_run-01_T2w.json
        │   └── sub-102_ses-01_run-01_T2w.nii.gz
        ├── fmap
        │   ├── sub-102_ses-01_dir-AP_epi.json
        │   └── sub-102_ses-01_dir-AP_epi.nii.gz
        ├── func
        │   ├── sub-102_ses-01_task-rs_run-01_bold.json
        │   └── sub-102_ses-01_task-rs_run-01_bold.nii.gz
        └── sub-102_ses-01_scans.tsv
And I added this to my fieldmap json file :"IntendedFor":["ses-01/func/sub-102_ses-01_task-rs_run-01_bold.nii.gz"]
When I run fmriprep V21.0.2 on the exactly same raw_data it does the filedmap correction.

**Comment by mgxd** [comment]
I'm not sure what is happening, but I can try reproducing this - I'll just need any the (non-identifying) metadata for the _epi and _bold files.

**Comment by oesteban** [comment]
Perhaps @manzouri could run something like:

```
docker --rm -it --entrypoint=/bin/bash -v $bids_root_dir:/data:ro nipreps/fmriprep:22.0.0 find /data/sub-102/ses-01 -type f
```

and check that the fieldmaps are indeed accessible from the process.

For ourselves, we could add printing out the subject(s)' data tree to the report when executed very verbose. Evidently, none of this resolves the problem, but would certainly help when debugging these issues.

**Comment by manzouri** [comment]
Thanks @oesteban , I tried to run the comand but getting this : 
/usr/bin/find: /usr/bin/find: cannot execute binary file
By the way as I mentioned before it works perfectly fine with V21.0.2.
Do you think you can resolve this issue so I can use the latest version soon?

**Comment by oesteban** [comment]
> Thanks @oesteban , I tried to run the comand but getting this : /usr/bin/find: /usr/bin/find: cannot execute binary file

That's weird. What do you get from:

```
docker --rm -it --entrypoint=/bin/bash -v $bids_root_dir:/data:ro nipreps/fmriprep:22.0.0 ls -lah /data/sub-102/ses-01/*
```
?


> By the way as I mentioned before it works perfectly fine with V21.0.2. Do you think you can resolve this issue so I can use the latest version soon?

To resolve the issue, we will first need to understand it so that a solution can be tried and perhaps resolve it. The feedback we have received thus far has not been enough for us to understand it. That's why we are asking further questions.

**Comment by manzouri** [comment]
@oesteban , And after second command 
docker run --rm -it --entrypoint="" -v $bids_root_dir:/data:ro nipreps/fmriprep:22.0.0 ls -lah /data/sub-102/ses-01/*
ls: cannot access '/data/sub-102/ses-01/*': No such file or directory

**Comment by mgxd** [comment]
@manzouri would you be able to share this subject / some phantom data with us? Perhaps that would help finally track this down

**Comment by oesteban** [comment]
> docker run --rm -it --entrypoint="" -v $bids_root_dir:/data:ro nipreps/fmriprep:22.0.0 ls -lah /data/sub-102/ses-01/*
> ls: cannot access '/data/sub-102/ses-01/*': No such file or directory

This is surprising, especially after seeing the output of the first command.



> @manzouri would you be able to share this subject / some phantom data with us? Perhaps that would help finally track this down

I agree this seems to be the best way to understand what's going on.

**Comment by manzouri** [comment]
@oesteban , without * at the end I get the following
total 24K
drwxr-xr-x 5 1001 1002 4.0K Sep 12 14:02 .
drwxr-xr-x 3 1001 1002 4.0K Sep  9 10:59 ..
drwxr-xr-x 2 1001 1002 4.0K Sep 12 14:02 anat
drwxr-xr-x 2 1001 1002 4.0K Sep 19 08:29 fmap
drwxr-xr-x 2 1001 1002 4.0K Sep 21 20:51 func
-rw-r--r-- 1 1001 1002  410 Apr 27 20:44 sub-102_ses-01_scans.tsv

**Comment by manzouri** [comment]
@mgxd ,I can  share another pilot data acquired from Siemens and another type of field map but same problem if that helps?

├── dataset_description.json
└── sub-pilot1
    └── ses-01
        ├── anat
        │   ├── sub-pilot1_ses-01_run-01_T1w.json
        │   └── sub-pilot1_ses-01_run-01_T1w.nii.gz
        ├── fmap
        │   ├── sub-pilot1_ses-01_run-01_magnitude1.json
        │   ├── sub-pilot1_ses-01_run-01_magnitude1.nii.gz
        │   ├── sub-pilot1_ses-01_run-01_magnitude2.json
        │   ├── sub-pilot1_ses-01_run-01_magnitude2.nii.gz
        │   ├── sub-pilot1_ses-01_run-01_phasediff.json
        │   └── sub-pilot1_ses-01_run-01_phasediff.nii.gz
        └── func
            ├── sub-pilot1_ses-01_task-suppression_run-01_bold.json
            └── sub-pilot1_ses-01_task-suppression_run-01_bold.nii.gz

**Comment by mgxd** [comment]
That seems like a different fieldmap, which won't help in this case - if you can't share or don't have any similar data, we can think of ways to better inform us on what is going on when searching for estimation methods and include it in a future release.

**Comment by manzouri** [comment]
@mgxd , @oesteban Please let me know how I can help or better inform you about this issue?

**Comment by manzouri** [comment]
Hi @mgxd , I saw that you were busy with new release just wonder if there has been any progress with this issue also?

**Comment by oesteban** [comment]
> @mgxd , Unfortunately I can't share this data and I don't have any similar from GE scanner but I am having the same problem with the fieldmaps from Siemens and I can easily share that data with you. Would be great if you can work on the available data to resolve this.

This would be the fastest way. If you are seeing the same issue on Siemens data and can share those, I'm fairly confident that's going to be the best option.

**Comment by manzouri** [comment]
@oesteban , would you please give me your email to share the link?

**Comment by oesteban** [comment]
nipreps@gmail.com should do ;)

thanks!

**Comment by manzouri** [comment]
thanks @oesteban  , just sent it and hope it helps :)

**Comment by oesteban** [comment]
Thanks @manzouri, I can replicate the issue. Will look into this today or tomorrow.

**Comment by manzouri** [comment]
Hi @oesteban , I wonder if you had some time to try my data and if you also have the same problem when running fmriprep from your end?
